pkg/cmd/server/origin/handlers.go

```diff
@@ -21,30 +21,32 @@ import (
 	"github.com/openshift/origin/pkg/authorization/authorizer"
 )
 
+// TODO once we have a MuxHelper we will not need to hardcode this list of paths
+// RootPaths are paths we server on from root
+var RootPaths kapi.RootPaths = kapi.RootPaths{Paths: []string{
+	"/api",
+	"/api/v1beta3",
```

Comment:
Probability of snark in this thread: 100%

On Thu, Aug 13, 2015 at 11:16 AM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/origin/handlers.go
> <https://github.com/openshift/origin/pull/4074#discussion_r36986219>:
>
> > @@ -21,30 +21,32 @@ import (
> >  	"github.com/openshift/origin/pkg/authorization/authorizer"
> >  )
> >
> > +// TODO once we have a MuxHelper we will not need to hardcode this list of paths
> > +// RootPaths are paths we server on from root
> > +var RootPaths kapi.RootPaths = kapi.RootPaths{Paths: []string{
> > +	"/api",
> > +	"/api/v1beta3",
>
> Probability of @deads2k <https://github.com/deads2k> asking for another
> integration test any given day... rising.
>
> Probability of anyone regretting a good test that ensures we stay in sync
> as we rebase on kube and update our master config: zero?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4074/files#r36986219>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift